src/core/modules/Nebula.hpp

```diff
@@ -98,7 +98,7 @@ friend class NebulaMgr;
 	Q_FLAG(TypeGroup)
 
 	//! A pre-defined set of specifiers for the catalogs filter
-	static constexpr CatalogGroup AllCatalogs = static_cast<CatalogGroup>(CatNGC|CatIC|CatM|CatC|CatB|CatSh2|CatLBN|CatLDN|CatRCW|CatVdB|CatCr|CatMel|CatPGC|CatUGC|CatCed|CatArp|CatVV|CatPK|CatPNG|CatSNRG|CatACO|CatHCG|CatESO|CatVdBH|CatDWB|CatTr|CatSt|CatRu|CatOther);
+	static constexpr CatalogGroup AllCatalogs = static_cast<CatalogGroup>(CatNGC|CatIC|CatM|CatC|CatB|CatSh2|CatLBN|CatLDN|CatRCW|CatVdB|CatCr|CatMel|CatPGC|CatUGC|CatCed|CatArp|CatVV|CatPK|CatPNG|CatSNRG|CatACO|CatHCG|CatESO|CatVdBH|CatDWB|CatTr|CatSt|CatRu|CatVdBHa|CatOther);
```

Comment:
Probably it's time to add a value Nebula::CatAll =0xFFFFFFFF to the CatalogGroupFlags. With this, you won't forget to OR all existing values.

Reply:
Maybe in version 4

Reply:
Huh? Just add this and remove the extra const AllCatalogs. Cannot be simpler. Or what did I overlook?